### DISCUSSION TITLE
Welcome to XploitSPY Discussions!

### DISCUSSION THREAD

**Discussion by RaoMK** [XploitWizer/XploitSPY#89] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tanveer187** [comment]
hello sir why can,t setup server plzz fix it

**Reply by RaoMK** [reply]
its working fine no issue found, make sure following steps properly

**Comment by PR0GR4M404** [comment]
![IMG_20210507_232337.jpg](https://user-images.githubusercontent.com/81549649/117480064-4f426b80-af8b-11eb-8f55-d0cbd4e37d7c.jpg)

I want to ask the next how?

**Reply by RaoMK** [reply]
Use a proper server with a static ip address

**Comment by codercracksachin** [comment]
Its not opening with my ip address

**Comment by moviedork** [comment]
i can't use the default username and password, any idea why?
